hyphae/src/myceli_api.rs

```diff
@@ -0,0 +1,118 @@
+use anyhow::{bail, Result};
+use messages::{
+    ApplicationAPI, DataProtocol, Message, MessageChunker, SimpleChunker, TransmissionBlock,
+};
+use std::net::{ToSocketAddrs, UdpSocket};
+use std::rc::Rc;
+use std::thread::sleep;
+use std::time::Duration;
+use tracing::{debug, info, warn};
+
+const MTU: u16 = 60;
```

Comment:
This could be a bit brittle if the MTU elsewhere is changed.  Maybe make a note in each invocation to sync them, or use an env var for the shared value?